src/main.jl

```diff
@@ -118,11 +89,10 @@ function shutdown_gracefully(c::Client)
         @warn "Unable to close client connection" ex
     end
     sleep(1)
-    exit(0)
+    return exit(0)
```

Comment:
The `exit` function never really returns since it will terminate Julia's VM. I would not change it here.